app/static/src/app/components/options/ParameterValues.vue

```diff
@@ -20,13 +20,23 @@ import {
 } from "vue";
 import { useStore } from "vuex";
 import { ModelMutation } from "../../store/model/mutations";
+import { Dict } from "../../types/utilTypes";
 
 export default defineComponent({
     name: "ParameterValues",
     setup() {
         const store = useStore();
-        const paramValues = computed(() => store.state.model.parameterValues);
-        const paramNames = computed(() => (paramValues.value ? Array.from(paramValues.value.keys()) as string[] : []));
+        const paramValuesMap = computed(() => store.state.model.parameterValues);
+        const paramNames = computed(
+            () => (paramValuesMap.value ? Array.from(paramValuesMap.value.keys()) as string[] : [])
+        );
+        const paramValues = computed(() => {
+            return paramNames.value.reduce((values: Dict<number>, key: string) => {
+                // eslint-disable-next-line no-param-reassign
+                values[key] = paramValuesMap.value.get(key)!;
+                return values;
+            }, {} as Dict<number>);
+        });
```

Comment:
Yeah, it's covered by the render tests as these are the values displayed. That lint suppress is just there because it doesn't like modifying a parmeter inside a method, but that's the easiest way to make this reduce work.